app/src/main/java/zapsolutions/zap/lnurl/Pair.java

```diff
@@ -0,0 +1,27 @@
+package zapsolutions.zap.lnurl;
+
+// clone of org.apache.commons.lang3.tuple.Pair;
+
+public class Pair<K, V> {
```

Comment:
I think we can also use `androidx.core.util.Pair`

Reply:
Yes, that makes sense, didn't know that exists. Thanks!